requirements.txt

```diff
@@ -2,4 +2,5 @@ aiohttp>=3.7.3
 dask>=2021.01.1
 distributed>=2021.01.1
 jinja2
-tornado>=5
\ No newline at end of file
+tornado>=5
+azure-identity
```

Comment:
These requirements are just the minimum for the package and not any of the providers. 

We use extra requirements to keep each cloud's dependencies separate.

So could you add it here instead?

https://github.com/dask/dask-cloudprovider/blob/f9c3272f833dad56101a9a8310039ac7cd5dc8a0/setup.py#L10-L15